java-vtl-script/src/test/java/no/ssb/vtl/script/functions/VTLSubstrTest.java

```diff
@@ -40,8 +40,7 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Override
-    public void testInvokeWithPositiveNumber() throws Exception {
+    public void testInvokeWithPositiveArgs() throws Exception {
```

Comment:
I still think this test method needs to be broken up.
And since the test name is inherited from the `VTLNumberFunctionTest` interface it does not really make sense here.
(Same applies to `testInvokeWithNegativeArgs` below)

Reply:
Agree. Drop the interface here, it is not helping. Breaking up the tests is important as well. Test one behaviour per method, use a relevant method name. Add descriptions (JAssert ones) where what you are asserting is not easy to grasp. And if you want to test several things, use Parameterized and/or SoftAssertions.